tests/plotting/test_plotting.py

```diff
@@ -4100,9 +4103,6 @@ def _generate_direction_object_functions() -> List[Tuple[str, FunctionType]]:
         'Sphere',
```

Comment:
```suggestion
        'SolidSphere', 
        'SolidSphereGeneric',
        'Sphere',
        'Text3D'
```

I should have re-added these here when I suggested removing them from the conditional 0.43 block

Thanks for your patience dealing with these changes!